### PR TITLE
Adding Int32 and Real64 ValueTypes parsing

### PR DESCRIPTION
**Pull request by gaddiab**
Adding two switch case statements to handle Values type "Int32 and Real64".

Attached is a sample evtx file that contains events that have both types in some events.

[Microsoft-Windows-AppReadiness.zip](https://github.com/Velocidex/evtx/files/7473616/Microsoft-Windows-AppReadiness.zip)

### CODE REVIEW

File: evtx.go [hunk 1]
```diff
@@ -375,6 +375,22 @@ func (self *ParseContext) ConsumeUint32() uint32 {
...
+
+	if len(self.buff) < self.offset+4 {
+		return 0
+	}
+
+	buf := bytes.NewReader(self.buff[self.offset:])
+	err := binary.Read(buf, binary.LittleEndian, &ret)
+	if err != nil {
+		fmt.Println("Binary read for Int32 Failed:", err)
+	}
```
Comment (williballenthin) [inline evtx.go:388]:
```suggestion
	if err != nil {
		return 0
	}
```
looks like a left-over debug statement? same with Real64 routine.

Comment (gaddiab) [reply evtx.go]: yes Thanks!

File: evtx.go [hunk 2]
```diff
@@ -385,6 +401,22 @@ func (self *ParseContext) ConsumeUint64() uint64 {
...
+
+	if len(self.buff) < self.offset+8 {
+		return 0
+	}
+
+	buf := bytes.NewReader(self.buff[self.offset:])
+	err := binary.Read(buf, binary.LittleEndian, &ret)
+	if err != nil {
+		fmt.Println("Binary read for Real64 Failed:", err)
+	}
```
Comment (williballenthin) [inline evtx.go:414]:
```suggestion
	if err != nil {
		return 0
	}
```

you should prefer feedback from the project owners - I'm just an innocent bystander :-)

Comment (scudette) [reply evtx.go:414]: Thank you for reviewing! I concur with your suggestions ☺️

**Comment by scudette** [comment]
Thanks!

**Pull request merged**